src/lib/optimizer/strategy/expression_reduction_rule.cpp

```diff
@@ -202,6 +202,11 @@ void ExpressionReductionRule::rewrite_like_prefix_wildcard(std::shared_ptr<Abstr
   }
 
   const auto multi_char_wildcard_pos = pattern.find_first_of('%');
+  // TODO(user): we do not rewrite LIKEs with multiple wildcards here. Theoretically, we could rewrite
```

Comment:
`TODO(anyone)` is the consensus. Also, in the browser this looks as if the comment does not use the full 120 characters per line.